### DISCUSSION TITLE
Error enabling rabbitmq_delayed_message_exchange on CentOS 7.9

### DISCUSSION THREAD

**Discussion by anboggia** [rabbitmq/rabbitmq-delayed-message-exchange#279] [Q&A]
Hi,
I have installed RabbitMQ 3.13 on a linux server (CentOS 7.9). Erlang version is 26.2.1.
I'm trying to enable the rabbitmq_delayed_message_exchange 3.13.0 through this steps:
1. download the .ez file with wget command
2. move the .ez file in /usr/lib/rabbitmq/lib/rabbitmq_server-3.13.0/plugins folder
3. run command 'rabbitmq-plugins enable rabbitmq_delayed_message_exchange' that raise following error

Output of step 3:
Enabling plugins on node rabbit@centoshost:
rabbitmq_delayed_message_exchange
The following plugins have been configured:
  rabbitmq_delayed_message_exchange
  rabbitmq_management
  rabbitmq_management_agent
  rabbitmq_web_dispatch
Applying plugin configuration to rabbit@centoshost...
Stack trace:

** (CaseClauseError) no case clause matching: {:badrpc, {:EXIT, {:aborted, {:node_not_running, :rabbit@centoshost}}}}
    (rabbitmqctl 3.13.0-dev) lib/rabbitmq/cli/plugins/plugins_helpers.ex:215: RabbitMQ.CLI.Plugins.Helpers.update_enabled_plugins/2
    (rabbitmqctl 3.13.0-dev) lib/rabbitmq/cli/plugins/plugins_helpers.ex:112: RabbitMQ.CLI.Plugins.Helpers.update_enabled_plugins/4
    (rabbitmqctl 3.13.0-dev) lib/rabbitmq/cli/plugins/commands/enable_command.ex:127: anonymous fn/6 in RabbitMQ.CLI.Plugins.Commands.EnableCommand.do_run/2
    (elixir 1.15.4) lib/stream.ex:1530: anonymous fn/2 in Stream.iterate/2
    (elixir 1.15.4) lib/stream.ex:1757: Stream.do_unfold/4
    (elixir 1.15.4) lib/stream.ex:1828: Enumerable.Stream.do_each/4
    (elixir 1.15.4) lib/stream.ex:1052: Stream.do_transform_inner_enum/7
    (elixir 1.15.4) lib/stream.ex:1828: Enumerable.Stream.do_each/4

{:case_clause, {:badrpc, {:EXIT, {:aborted, {:node_not_running, :rabbit@centoshost}}}}}

4. We checked the rabbit node status with command 'rabbitmqctl status' and everything seems fine 

Could someone help understanding this issue?
Thank you

**Comment by mkuratczyk** [comment]
It's been two years since we stopped any support for CentOS 7.
https://www.rabbitmq.com/blog/2022/04/26/centos-7-support-discontinued
